examples/simple-platform/run.adoc

```diff
@@ -0,0 +1,109 @@
+= Run the Example Locally
+
+This section describes how to start all four apps that are used in this
+example.
+
+NOTE: The following procedures assume that you are continuing from
+link:configure.adoc[Configure the Example].
+
+
+== Build and start the `platform-deploy` node
+
+. Open a new terminal and run the following commands from the `interbit`
+  repository's root:
++
+[source,bash]
+----
+npm i
+source secrets/platform-deploy.sh
+cd packages/platform-deploy
+npm run build:dev
+npm start
+----
++
+Our package script runs a post-install step which builds all of the
+modules in the `interbit` repository. The `build:dev` step for
+`platform-deploy` creates a new manifest file using the newly-generated
+keys and hashed covenant files. The build step also updates the chain
+IDs for the other modules, including `app-account` and
+`interbit-template`.
++
+The `start` command spins up the `platform-deploy` node, which listens
+for incoming connections from the `web-auth-endpoint` node and the
+Account and Template apps' browser nodes.
++
+IMPORTANT: Leave this terminal open.
+
+. Wait 3 minutes.
```

Comment:
It might be a good idea to include a sample of what's printed to the terminal when the chains have started up correctly. I *think* this is when you see `Applying covenant XXXX to templateControl (YYYY) JOINING....` for example.